deployments/cdi/systemd/cdi-spec.service

```diff
@@ -0,0 +1,12 @@
+[Unit]
```

Comment:
this is just an example, can you add some comments explaining this ?
or better yet add a README here for it.

in actual deployment you would want also to have an "After" key as this would need to run after the service which configures SR-IOV.